models/engine/file_storage.py

```diff
@@ -68,3 +68,20 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """retrieve one object"""
+        if cls.__name__ in classes and id is not None:
+            objs = self.all(cls.__name__)
+            for key in objs.keys():
+                if key == cls.__name__ + '.' + id:
+                    return objs[key]
+        return None
```

Comment:
You can optimize all this part +74 to +79 using the .`get(key)` dictionary function, where `key ` is class name + object id

```suggestion

        def get(self, cls, id):
            clss = self.all(cls)
            key = cls + '.' + id
            return clss.get(key)
```

Reply:
Excellent, I already update it with another change: I had to add the `__name__` special variable to `cls` as you can see below and now it works for me.

`key = cls.__name__ + '.' + id`

Reply:
I reviewed, but the cls is already an string. So,  `cls.__name__` is not necessary. 
Error:
```
File "/home/vagrant/AirBnB_clone_v3/models/engine/db_storage.py", line __, in **count**
    if cls.__name__ in classes:
AttributeError: 'str' object has no attribute '_**_name__**'
```